homeassistant/components/zha/__init__.py

```diff
@@ -285,6 +252,34 @@ def device_removed(self, device):
         if device.ieee in self._events:
             self._events.pop(device.ieee)
 
+    def get_device_entity(self, ieee_str):
+        """Return ZHADeviceEntity for given ieee."""
+        ieee = self.convert_ieee(ieee_str)
+        if ieee in self._device_registry:
+            entities = self._device_registry[ieee]
+            entity = next(
+                ent for ent in entities if isinstance(ent, ZhaDeviceEntity))
+            return entity
+        return None
+
+    def get_entities_for_ieee(self, ieee_str):
+        """Return list of entities for given ieee."""
+        ieee = self.convert_ieee(ieee_str)
+        if ieee in self._device_registry:
+            return self._device_registry[ieee]
+        return []
+
+    @staticmethod
+    def convert_ieee(ieee_str):
+        """Convert given ieee string to EmberEUI64."""
+        from bellows.types import EmberEUI64, uint8_t
```

Comment:
should we use `zigpy.types.EUI64` instead? This seems to be a remnant from time before bellows was split into zigpy and bellows.

Reply:
I'll take a look